File: algorithm/26. Remove Duplicates from Sorted Array/26.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if(nums.size()==0)
            return 0;
        int result=0;
        for (int i=0;i<nums.size();i++)
        {
            if(nums[result]!=nums[i])
                nums[++result]=nums[i];
        }
        return result+1;
    }
};

File: algorithm/26. Remove Duplicates from Sorted Array/26.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if(len(nums)==0):
            return 0
        result=0
        for i in xrange(len(nums)):
            if nums[result]!=nums[i]:
                result+=1
                nums[result]=nums[i]
        return result+1
